chore: added general requirements for 3.4 (#1913)
mendersoftware/mender-docs: 02.Overview/16.Requirements/docs.md

---
title: Requirements
taxonomy:
    category: docs
---

## Working with Firewalls

> Corporate Firewalls must permit the following URLs to the devices in order for Mender to work correctly:

Reviewer: ```suggestion
The following URLs and access types need outgoing permissions in firewalls in order for Hosted Mender to work correctly:
```

(copied over from #1913 review)

Author: Done

> Corporate Firewalls must permit the following URLs to the devices and for letting Mender works correctly:

Reviewer 2: ```suggestion
Corporate Firewalls must permit the following URLs to the devices in order for Mender to work correctly:
```

**Mender access**: Devices, APIs and browser access

> * `https://mender.example.com` - It's the same address you specify it in the [Server Installation Section](../../07.Server-installation/)

Reviewer: ```suggestion
* `https://mender.example.com` - Use the same address you specify it in the [Server Installation Section](../../07.Server-installation/)
```

**Artifact storage access**: Devices, APIs and browser access

[ui-tabs position="top-left" active="0" theme="default" ]
[ui-tab title="Amazon S3"]

> If you use Amazon S3 for storing artifacts, you have to permit access to the following URLs:

Reviewer: ```suggestion
Permit access to the following URLs:
```

Author: Done

* `https://s3.amazonaws.com/<my-example-artifact-storage>` - Amazon S3 Path-style request
* `https://<my-example-artifact-storage>.s3.amazonaws.com` - Amazon S3 Virtual-hosted-style request
[/ui-tab]
[ui-tab title="Minio"]
If you use Minio for storing artifacts, you have to permit access to your 
Minio URL defined in [Minio Setup Section](../../07.Server-installation/04.Production-installation-with-kubernetes/04.Minio/docs.md)
* `https://artifacts.example.com`
[/ui-tab]
[/ui-tabs]

## Amazon S3 IAM policies

> If you are using Amazon S3 bucket for storing Mender Artifacts, you can use the following minimum policy set:

Reviewer: ```suggestion
A minimum policy set to use an Amazon S3 bucket to store Mender Artifacts is:
```
